test/foundry/unit/StakingRewardsV2/StakingRewardsV2.t.sol

```diff
@@ -1077,30 +1024,12 @@ contract StakingRewardsV2Test is DefaultStakingV2Setup {
         assertEq(initialTotalSupply - 1 weeks, finalTotalSupply);
     }
 
-    function test_unstakeEscrowSkipCooldown_Does_Change_totalSupply() public {
```

Comment:
**Though:** Can we have an equivalent of this test with `unstakeEscrowAdmin`?

Reply:
Added `test_unstakeEscrowAdmin_Does_Change_totalSupply`